src/state/types.ts

```diff
@@ -256,6 +256,89 @@ export interface PredictionsState {
   bets: BetData
 }
 
+// Voting
+
+/* eslint-disable camelcase */
+/**
+ * @see https://hub.snapshot.page/graphql
+ */
+export interface VoteWhere {
```

Comment:
At some point should we maybe consider breaking these `/state/types` into separate files, by view? Else there's no upper limit to how big this puppy could get.